src/main/app/utils/progress-bar-utils.ts

```diff
@@ -66,17 +66,26 @@ const buildProgressBarStages = (userCase: Partial<CaseWithId>): object => {
   return progressBarStages;
 };
 
-const isFinalOrderActive = (userCase: Partial<CaseWithId>) => {
-  console.log(JSON.stringify(userCase.selectTypeOfOrder));
+const isFinalOrderIssued = (userCase: Partial<CaseWithId>) => {
   if (userCase.selectTypeOfOrder === SelectTypeOfOrderEnum.finl) {
     return true;
   }
   return false;
 };
 
-const iscaseClosedActive = (userCase: Partial<CaseWithId>) => {
-  console.log(JSON.stringify(userCase.doesOrderClosesCase));
-  if (userCase.doesOrderClosesCase === YesOrNo.YES) {
+const isHearingOrderActive = (userCase: Partial<CaseWithId>) => {
+  if (
+    userCase.orderCollection ||
+    userCase.state === State.DECISION_OUTCOME ||
+    userCase.state === State.PREPARE_FOR_HEARING_CONDUCT_HEARING
+  ) {
+    return true;
+  }
+  return false;
+};
+
+const isResponseSubmitted = (userCase: Partial<CaseWithId>) => {
+  if (userCase.citizenResponseC7DocumentList) {
```

Comment:
How do we know the response is submitted by the one who logged in, if there are multiple respondents